app/models/legislator.rb

```diff
@@ -61,7 +61,7 @@ class Legislator < ActiveRecord::Base
   scope :current_supporters,-> { joins(sponsorships: :bill).where('sponsorships.id IS NOT NULL').distinct.merge(Bill.current) }
   scope :allowed_states,-> {
     joins({district: :state}, :state)
-    .where('state.abbrev IS IN (:state_abbrev)', state_abbrev: ALLOWED_STATES) }
+    .where('states.abbrev IN (:state_abbrev)', state_abbrev: ALLOWED_STATES) }
```

Comment:
Expression at 64, 78 should be on its own line.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.